tests/FunctionalTests/ConversationTest.php

```diff
@@ -52,12 +52,13 @@ public function testAll()
         $this->assertSame(['Hello', 'World'], $conversation->getWelcomeMessages());
         $custom = $conversation->getCustom();
         $this->assertFalse(isset($custom['test']) && $randomTestString === $custom['test']);
-        $this->assertSame(['my_user' => ['notify' => true, 'access' => 'ReadWrite']], $conversation->getParticipants());
+        $this->assertEquals(['my_user' => ['notify' => true, 'access' => 'ReadWrite']], $conversation->getParticipants());
         $this->assertInstanceOf(\DateTimeImmutable::class, $conversation->getCreatedAt());
 
         $response = $this->api->createOrUpdate($conversationId, [
-            'participants' => ['my_user'],
-            'subject' => 'An amazing conversation',
+            'subject' => 'An amazing conversation!',
+            'welcomeMessages' => ['Hello', 'World!'],
+            'photoUrl' => 'another_photo_url',
```

Comment:
Since I removed the second test below from our comments, I simply moved those properties them here.